doc/source/getting_started/index.rst

```diff
@@ -664,9 +666,6 @@ material is enlisted in the community contributed :ref:`tutorials`.
 
     install
     overview
-    10min
     intro_tutorials/index
-    basics
-    dsintro
```

Comment:
I don't think we should just remove basics and dsintro, but keep them somewhere in the docs. I personally would move them to the user guide, but so that means they need to be added to the toctree there if we want to do that.

Reply:
I added them as links in the user guide but hadn't added it to the toctree there yet.

 https://github.com/pandas-dev/pandas/pull/32389/files#diff-e81313c8de3a67eb46e6744b7c895af2R15